pkg/properties/propertiesGet_test.go

```diff
@@ -0,0 +1,442 @@
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `namespace      name                     value
+validNamespace aPrefix.property.aSuffix prefixSuffixVal
+
+Total:1
+`
+
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+func TestValidNamespaceWithNoMatchingPrefixAndSuffixReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := ""
+	prefix := "noMatchingPrefix"
+	suffix := "noMatchingSuffix"
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `Total:0
+`
+
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+
+func TestValidNamespaceAndNoMatchingPrefixReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := ""
+	prefix := "noMatchingPrefix"
+	suffix := ""
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `Total:0
+`
+
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+func TestValidNamespaceAndNoMatchingSuffixReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := ""
+	prefix := ""
+	suffix := "noMatchingSuffix"
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `Total:0
+`
+
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+
+// namespace + name
+func TestValidNameWithValidNameReturnsOk(t *testing.T) {
+	//Given...
+	namespace := "validNamespace"
+	name := "property0"
+	prefix := ""
+	suffix := ""
+	propertiesOutputFormat := "summary"
+
+	server := NewPropertiesServletMock(t)
+	apiServerUrl := server.URL
+	defer server.Close()
+
+	console := utils.NewMockConsole()
+	expectedOutput := `namespace      name      value
+validNamespace property0 value0
+
+Total:1
+`
+	//When
+	err := GetProperties(namespace, name, prefix, suffix, apiServerUrl, propertiesOutputFormat, console)
+
+	//Then
+	assert.Nil(t, err)
+	assert.Equal(t, expectedOutput, console.ReadText())
+}
+
+func TestValidNameWithEmptyValueValidNameReturnsOk(t *testing.T) {
```

Comment:
This test should return the property with the empty value. Please see example from local api below:

```shell
 cli % ./bin/galasactl-darwin-x86_64 properties get --namespace framework --name test.empty.stream --bootstrap htpps://127.0.0.1/bootstrap
namespace name              value
framework test.empty.stream 

Total:1 
```